packages/ui/src/validators/index.ts

```diff
@@ -1,15 +1,18 @@
-import { AuthFormData, Validator } from '../types';
+import { AuthFormData, PasswordSettings, Validator } from '../types';
 import isEmpty from 'lodash/isEmpty';
 import merge from 'lodash/merge';
 
 // Runs all validators given. Resolves if there are no error. Rejects otherwise.
 export const runValidators = async (
   formData: AuthFormData,
   touchData: AuthFormData,
+  passwordSettings: PasswordSettings,
   validators: Validator[]
 ) => {
   const errors = await Promise.all(
-    validators.map((validator) => validator(formData, touchData))
+    validators.map((validator) =>
+      validator(formData, touchData, passwordSettings)
```

Comment:
Now I'm adding in the optional passwordSettings to each validator if it needs it.